client/src/containers/filters/years-range/component.tsx

```diff
@@ -108,7 +108,7 @@ export const YearsRangeFilter: React.FC<YearsRangeFilterProps> = ({
         {...getReferenceProps({
           ref: reference,
         })}
-        className="relative w-full py-2 pl-3 pr-10 text-left bg-white border-0.5 min-h-[2.5rem] border-gray-300 rounded-md shadow-sm cursor-pointer focus:outline-none focus:ring-1 focus:ring-green-700 focus:border-green-700 sm:text-sm"
+        className="relative w-full py-2 pl-3 pr-10 text-left bg-white border min-h-[2.5rem] border-gray-300 rounded-md shadow-sm cursor-pointer focus:outline-none focus:ring-1 focus:ring-green-700 focus:border-green-700 sm:text-sm"
```

Comment:
Why this border? It's much bigger than the others
<img width="367" alt="image" src="https://user-images.githubusercontent.com/1385934/178959591-5a45c850-fd50-4101-afce-4d7d2af23f21.png">

Reply:
in the select is missing the ring, on it